### PR TITLE
labels: add subsystem label for API doc

### PR DESCRIPTION
**Pull request by mscdex**
This is only done if a PR is *only* modifying a single subsystem's API documentation.

Feel free to reject this and implement it some other way if you don't like how I did this, I just figured it'd be better to submit some kind of PR instead of just creating a feature request issue.

### CODE REVIEW

File: test/node-js-subsystem-labels.test.js [hunk 1]
```diff
@@ -130,3 +130,13 @@ tap.test('label: lib internals without "lib / src" limiting', (t) => {
 
   t.end()
 })
+
+tap.test('label: add subsystem when ./doc/api/<subsystem>.md has been changed', (t) => {
+  const labels = nodeLabels.resolveLabels([
+    'doc/api/fs.md'
+  ], false)
+
+  t.same(labels, ['doc', 'fs'])
```
Comment (phillipj) [inline test/node-js-subsystem-labels.test.js]: What labels should be added if `./doc/api/fs.md` *and* `./doc/api/http.md` had been changed in the same PR?

Comment (mscdex) [inline test/node-js-subsystem-labels.test.js]: Just `doc`.

Comment (mscdex) [inline test/node-js-subsystem-labels.test.js]: I just added a test to explicitly test that scenario.

Comment (phillipj) [inline test/node-js-subsystem-labels.test.js]: > I just added a test to explicitly test that scenario.

Perfect 👍

**Comment by phillipj** [comment]
LGTM. Thumbs up for doing a PR rather than an issue :)

**Comment by phillipj** [comment]
@mscdex just deployed

**Comment by mscdex** [comment]
aaaand it works! https://github.com/nodejs/node/pull/7361#event-700067634

**Pull request merged**